src/json-schema.ts

```diff
@@ -235,7 +235,25 @@ export interface Schema<
   const?: Const; // 💪
   enum?: Enum; // 💪
   type?: Type; // 💪
-  format?: Type extends "string" ? string : never; // ⚠️ only makes sense for string types
+  format?: Type extends "string"
```

Comment:
Can you add a comment referencing where these come from? Assuming this is in the spec, lgtm

Reply:
I can add later on in my alt account, good call. It's actually not from the spec but from `ajv`; they don't have a type exposing their default supported formats, unfortunately.

Reply:
That's fine; since we're using ajv under the hood let's just point to wherever it's being implemented.